stream_alert/athena_partition_refresh/main.py

```diff
@@ -403,7 +383,7 @@ class StreamAlertSQSClient(object):
         received_messages: A list of receieved SQS messages
         processed_messages: A list of processed SQS messages
     """
-    QUEUENAME = 'streamalert_athena_data_bucket_notifications'
+    DEFAULT_QUEUE_NAME = '{}_streamalert_athena_data_bucket_notifications'
```

Comment:
This is a pretty long Queue name, can we shorten it? The max length is 80 chars.

Reply:
Yeah we can - I was just mimicking what our queue name is internally